starter-code/scripts/articles.js

```diff
@@ -3,22 +3,40 @@ var articles = [];
 function Article (opts) {
   // TODO: Use the object passed in to complete this constructor function:
   // Save ALL the properties of `opts` into `this`.
-  this.author = opts.author;
+  this.author      = opts.author;
+  this.title       = opts.title;
+  this.category    = opts.category;
+  this.authorUrl   = opts.authorUrl;
+  this.publishedOn = opts.publishedOn;
+  this.body        = opts.body;
 }
 
+var firstArticle = ourLocalData[0];
```

Comment:
Was this added for testing? Feel free to remove these three lines. Line 58 will create the new Articles for you.